src/main/java/com/oltpbenchmark/types/DatabaseType.java

```diff
@@ -45,9 +45,9 @@ public enum DatabaseType {
     ORACLE(true, false),
     SINGLESTORE(true, false),
     SPANNER(false, true),
-    SQLAZURE(true, false),
+    SQLAZURE(true, true),
     SQLITE(true, false),
-    SQLSERVER(true, false),
+    SQLSERVER(true, true),
```

Comment:
Need to include the column names in the `INSERT` statement for the `SET INSERT_IDENTITY [table_name] ON` command to work for the session.